projects/schematics/src/migrations/migrations.json

```diff
@@ -46,6 +46,11 @@
       "description": "Migrate ssr to the new setup"
     },
 
+    "00-migration-v4-rename-symbol": {
```

Comment:
Since the schematics are executed in the alphabetical order for a given "version", what about introducing a consistent naming convention:`migration-v4-<number>-<whatever-meaningful-here>` ? For example:
```
migration-v4-00-rename-symbol
migration-v4-01-removed-public-api-deprecation
...
```

Reply:
Agreed. Let's do it in a separate ticket, after this one is merged.